### DISCUSSION TITLE
Issue Creation w/ Template Broken

### DISCUSSION THREAD

**Discussion by haIIux** [community/community#108619] [Mobile]
While using GitHub mobile using the below details, I attempted to create a new issue and it pushed me to GitHub web, and 404'd. 

- Navigate to repository with known Issue Templates. 
- Press + to add a new issue & select a template. 
- Note: It pushes me to my repository URL with the endpoint /issues/new?template=FEATURE_REQUEST.yaml

Please let me know if there is any more details needed. 
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.149.0 (122782244)
Device: iPhone 15 Pro (iOS 17.2.1)
</details>

**Comment by iskae** [comment]
Hi @hallux 👋 

Thank you for reaching out ❤️ 

I _assume_ the template you are selecting is an issue form, right? If that is the case, unfortunately we are not supporting issue forms _yet_ 😅 You should be able to use regular templates though 🙇 

In any case, I have added your request to our internal tracking issue for issue forms 👍 Please stay tuned for our future releases 😊

**Reply by haIIux** [reply]
Hi @iskae!

Possibly? I've attached a screenshot of what I see when I attempt to use a template. Works fine if I click "No Template", but any of those templates cause the issues. 

![image](https://github.com/community/community/assets/47156971/4c185c0d-7e6f-4cb2-ae50-098c99539869)

**Comment by TPS** [comment]
https://github.com/orgs/community/discussions/118586?
